src/test/fuzz/key.cpp

```diff
@@ -305,4 +306,25 @@ FUZZ_TARGET_INIT(key, initialize_key)
             assert(key == loaded_key);
         }
     }
+
+    {
+        std::array<uint8_t, 32> rnd32;
+        memcpy(rnd32.data(), &random_uint256, 32);
+        assert(pubkey.EllSqEncode(rnd32)->size() == ELLSQ_ENCODED_SIZE);
+    }
+}
+
+FUZZ_TARGET_INIT(ellsq, initialize_key)
+{
+    if(buffer.size() < 64) {
+        return;
+    }
+
+    auto ellsq_bytes = buffer.first(64);
```

Comment:
You could do a stronger test here: start with a pubkey drawn from the fuzz data, and if it is valid, elligator encode it (with additional randomness drawn from the fuzz data), decode it, and see if it matches the original pubkey.

Reply:
Thanks. Done!